ga4gh/datamodel/reads.py

```diff
@@ -5,3 +5,57 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
+
+import os
+import glob
+
+import pysam
+
+import ga4gh.protocol as protocol
+
+
+class ReadGroupSet(object):
+    """
+    TODO document.
+    """
+    def __init__(self, id_, dataDir):
+        self._id = id_
+        self._dataDir = dataDir
+        self._readGroups = []
+        for fileType in ["sam", "bam"]:
+            pattern = "*.{}".format(fileType)
+            for path in glob.glob(os.path.join(self._dataDir, pattern)):
+                filename = os.path.split(path)[1]
+                localId = filename.split(".")[0]
+                readGroupId = "{}:{}".format(self._id, localId)
```

Comment:
I think so - we're not trying to support older versions of Python, and this form has been supported in 2.7 for a while now.